src/custom/components/CowBalance/index.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components/macro'
+import { Trans } from '@lingui/macro'
+import { CurrencyAmount, Currency } from '@uniswap/sdk-core'
+
+import { AMOUNT_PRECISION } from 'constants/index'
+import { ClaimSummaryTitle, ClaimTotal, ClaimSummary as ClaimSummaryWrapper } from 'pages/Claim/styled'
+import { formatMax, formatSmartLocaleAware } from 'utils/format'
+import CowProtocolLogo from 'components/CowProtocolLogo'
+
+const Wrapper = styled(ClaimSummaryWrapper)`
+  border-radius: 100px;
+`
+
+export interface CowBalanceProps {
+  account: string
+  balance?: CurrencyAmount<Currency>
+  title?: string
+}
+
+const CowBalance = ({ account, balance, title }: CowBalanceProps) => {
+  return (
+    <Wrapper>
+      <CowProtocolLogo size={100} />
+
+      {title && (
+        <ClaimSummaryTitle>
+          <Trans>{title}</Trans>
+        </ClaimSummaryTitle>
+      )}
+
+      {balance && !account && (
```

Comment:
Why `!account`?

Ahh ok, there's a comment in the PR body.

Can you add a TODO or comment to be explicit?